### PR TITLE
Cleanup log messaging

### PR DESCRIPTION
**Pull request by gabrieljackson**
This change corrects a few logging issues:

 * Standardized formatting errors with '%v'.
 * Standardized failure warning word usage.
 * Corrected an instance of using the standard log library when
   the gitea log library should be used instead.

This targets issue https://github.com/go-gitea/gitea/issues/775

### CODE REVIEW

File: models/migrations/migrations.go [hunk 1]
```diff
@@ -328,7 +328,7 @@ func attachmentRefactor(x *xorm.Engine) error {
 
 		dumpPath := path.Join(setting.LogRootPath, "attachment_path.dump")
 		ioutil.WriteFile(dumpPath, buf.Bytes(), 0666)
-		fmt.Println("Fail to rename some attachments, old and new paths are saved into:", dumpPath)
+		fmt.Println("Failed to rename some attachments, old and new paths are saved into:", dumpPath)
```
Comment (bkcsoft) [inline models/migrations/migrations.go]: Migrations run just like anything else so they shold probably use `log.Printf` like all the others

File: models/repo.go [hunk 2]
```diff
@@ -151,19 +151,19 @@ func NewRepoContext() {
 			// ExitError indicates this config is not set
 			if _, ok := err.(*exec.ExitError); ok || strings.TrimSpace(stdout) == "" {
 				if _, stderr, gerr := process.GetManager().Exec("NewRepoContext(set "+configKey+")", "git", "config", "--global", configKey, defaultValue); gerr != nil {
-					log.Fatal(4, "Fail to set git %s(%s): %s", configKey, gerr, stderr)
+					log.Fatal(4, "Failed to set git %s(%s): %s", configKey, gerr, stderr)
 				}
 				log.Info("Git config %s set to %s", configKey, defaultValue)
```
Comment (bkcsoft) [inline models/repo.go]: Infof ;)

File: modules/ssh/ssh.go [hunk 3]
```diff
@@ -169,12 +169,12 @@ func Listen(host string, port int) {
...
 
 		if err := os.MkdirAll(filePath, os.ModePerm); err != nil {
-			log.Error(4, "Fail to create dir %s: %v", filePath, err)
+			log.Error(4, "Failed to create dir %s: %v", filePath, err)
 		}
 
 		_, stderr, err := com.ExecCmd("ssh-keygen", "-f", keyPath, "-t", "rsa", "-N", "")
 		if err != nil {
-			panic(fmt.Sprintf("Fail to generate private key: %v - %s", err, stderr))
+			panic(fmt.Sprintf("Failed to generate private key: %v - %s", err, stderr))
```
Comment (bkcsoft) [inline modules/ssh/ssh.go]: Maybe just use `log.Fatal` (which does `os.Exit(1)` at the end)

**Review by bkcsoft — CHANGES_REQUESTED** [review]
Few things before LGTM :)

File: modules/ssh/ssh.go (hunk 3 above)
Comment (gabrieljackson) [reply modules/ssh/ssh.go]: Good catch. I didn't even notice the failure message was wrapped in a panic call. I removed two other uses of `panic()` in the latest commit.

**Comment by lunny** [comment]
LGTM

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by lunny** [comment]
@bkcsoft need your confirm and approve.

**Review by bkcsoft — APPROVED** [review]

**Pull request merged**